tests/test_raster.py

```diff
@@ -1,97 +1,125 @@
-
-def test_raster_metadata_minmax_no_id(raster_metadata):
-    req = Request.blank('?request=INVALID_REQUEST_TYPE&id=pr-tasmax-tasmin_day_BCSD-ANUSPLIN300-CanESM2_historical-rcp26_r1i1p1_19500101-21001231&var=tasmax')
-    resp = req.get_response(raster_metadata)
-    assert resp.status == '400 Bad Request'
-
-def test_raster_metadata_minmax_no_id(raster_metadata):
-    req = Request.blank('?request=GetMinMax&var=tasmax')
-    resp = req.get_response(raster_metadata)
-    assert resp.status == '400 Bad Request'
-
-def test_raster_metadata_minmax_no_var(raster_metadata):
-    req = Request.blank('?request=GetMinMax&id=pr-tasmax-tasmin_day_BCSD-ANUSPLIN300-CanESM2_historical-rcp26_r1i1p1_19500101-21001231')
-    resp = req.get_response(raster_metadata)
-    assert resp.status == '400 Bad Request'
-
-def test_raster_metadata_minmax_bad_id(raster_metadata):
-    req = Request.blank('?request=GetMinMax&id=NOT_A_VALID_ID&var=tasmax')
-    resp = req.get_response(raster_metadata)
-    assert resp.status == '404 Not Found'
+def test_ensemble_1_files(mm_test_session, ensemble_1, ensemble_1_data_files):
+    result = ensemble_files(mm_test_session, ensemble_1.name)
+    assert result == {df.unique_id: df.filename for df in ensemble_1_data_files}
+
+
+def test_ensemble_2_files(mm_test_session, ensemble_2, ensemble_2_data_files):
+    result = ensemble_files(mm_test_session, ensemble_2.name)
+    assert result == {df.unique_id: df.filename for df in ensemble_2_data_files}
+
+
+@pytest.mark.parametrize('root_url', ['foo/', 'bar/'])
+def test_db_raster_catalog(
+    mm_test_session, ensemble_1, ensemble_1_data_files, root_url
+):
+    result = db_raster_catalog(mm_test_session, ensemble_1.name, root_url)
+    assert result == {
+        df.unique_id: '{}{}'.format(root_url, basename(df.filename))
+        for df in ensemble_1_data_files
+    }
+
+
+@pytest.mark.parametrize(
+    'name, version, api_version, ensemble, root_url, handlers',
+    [
+        ('A name', 'Version 0.0.0.1', '0.0', 'ensemble_1', 'http://root.ca/',
+         [
+             {'url': 'data_file_1.nc', 'file': '/storage/data_file_1.nc'},
+             {'url': 'data_file_2.nc', 'file': '/storage/data_file_2.nc'},
+         ]),
+])
+def test_db_raster_configurator(
+    mm_test_session, name, version, api_version, ensemble, root_url, handlers,
+):
+    result = db_raster_configurator(
+        mm_test_session, name, version, api_version, ensemble, root_url,
+    )
+    assert all(
+        x in result.items()
+        for x in {
+            'root_url': root_url,
+            'name': name,
+            'version': version,
+            'ensemble': ensemble,
+            'api_version': api_version,
+        }.items()
+    )
+    assert all(
+        x in result['handlers']
+        for x in handlers
+    )
+
+
+@pytest.mark.parametrize('url', ['', '/url/is/irrelevant'])
+@pytest.mark.parametrize('root_url', [
+    'http://root.ca/',
```

Comment:
Not a big deal, but the [IETF reserves `example.com`](https://en.wikipedia.org/wiki/Example.com) for usages like this.